Q14/Codeville.rb

```diff
@@ -0,0 +1,18 @@
+def solution(weights, limit)
+    # p weights.sort!
+    boats = 0
+    check = 0
+    index = 0
+  
+    weights.each do |x|
```

Comment:
Is weights[index] not just 'x'?

Reply:
When I use x to cycle through the array, It gives me a cannot coerce nil to integer message